core/core_bind.cpp

```diff
@@ -1060,6 +1064,7 @@ void Geometry3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("segment_intersects_convex", "from", "to", "planes"), &Geometry3D::segment_intersects_convex);
 
 	ClassDB::bind_method(D_METHOD("clip_polygon", "points", "plane"), &Geometry3D::clip_polygon);
+	ClassDB::bind_method(D_METHOD("tetrahedralize_delaunay", "points"), &Geometry3D::tetrahedralize_delaunay);
```

Comment:
Can't return `Vector<uint32_t>` for a bind, it doesn't have a mapped type, needs to be `Vector<int32_t>`

Reply:
Thanks for the heads up, should be fixed now and once it compiles on my end I'll submit the commit.

Reply:
Compilation goes through all the way to the linking process and then throws this error:

- core\libcore.windows.editor.x86_64.a(core_bind.windows.editor.x86_64.o):core_bind.cpp:(.rdata$.refptr._ZN9core_bind10Geometry3D23tetrahedralize_delaunayERK6VectorI7Vector3E[.refptr._ZN9core_bind10Geometry3D23tetrahedralize_delaunayERK6VectorI7Vector3E]+0x0): undefined reference to `core_bind::Geometry3D::tetrahedralize_delaunay(Vector<Vector3> const&)`